.evergreen/ocsp/mock-ocsp-responder-requirements.txt

```diff
@@ -4,5 +4,6 @@ flask==2.2.5
 itsdangerous==2.1.2
 Jinja2==3.1.5
 MarkupSafe==2.1.4
-git+https://github.com/wbond/oscrypto.git@d5f3437
```

Comment:
@blink1073: This removal is what appears to have caused the regression in PHPC. I noticed that this commit was mentioned in https://github.com/wbond/oscrypto/issues/78#issuecomment-1756601738 as a workaround for the upstream issue with oscrypto 1.3.0.

The original context for this change was https://github.com/blink1073/drivers-evergreen-tools/commit/4483bcca6e9084d124632d277ed45561f2f7464e, but it looks like we'd do well to include a comment here for future reference. Might be a good idea to reference the upstream issue.

Reply:
Hmm, I had removed the specific tag because I was seeing a warning from pip.  I'll revert to use the tag.